torch/csrc/jit/passes/quantization.cpp

```diff
@@ -40,39 +40,45 @@ int groups, bool benchmark, bool deterministic, bool cudnn_enabled) -> Tensor"};
   return quantnodeLookup.find(n);
 }
 
-void insertQuantNodeParams(Node* quant, std::tuple<float, int> qparam) {
+void insertQuantNodeParams(Node* quant,
```

Comment:
Please run clang-format on your PR (and install clang-format hooks so that you don't have to do it manually every time).